tests/test-heap.cpp

```
@@ -144,7 +144,7 @@ ComPtr<IBuffer> createBuffer(IDevice* device, uint32_t size)
     return buffer;
 }
 
-GPU_TEST_CASE("heap-create", CUDA | Vulkan)
+GPU_TEST_CASE("heap-create", CUDA)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DOCTEST_GLOBAL_NO_WARNINGS is a macro then please configure it.

(unknownMacro)